decision/policies.py

```diff
@@ -210,3 +210,95 @@ def __init__(self):
 
     def get_best_action(self, world, vehicle):
         return classes.Action([], [], [], 0)
+
+
+class RebalancingPolicy(Policy):
+    def __init__(self):
+        super().__init__(0, 0)
+
+    def get_best_action(self, world, vehicle):
+        vehicle_has_scooter_inventory = len(vehicle.scooter_inventory) > 0
+        if vehicle.is_at_depot():
+            scooters_to_deliver = []
+            scooters_to_pickup = []
+            number_of_scooters_to_pick_up = 0
+            number_of_scooters_to_swap = 0
+            scooters_to_swap = []
+        else:
+            # If vehicle has scooter inventory, deliver all scooters and swap all swappable scooters
+            if vehicle_has_scooter_inventory:
+                # Deliver all scooters in scooter inventory, and don't pick up any new scooters
+                scooters_to_deliver = [
+                    scooter.id for scooter in vehicle.scooter_inventory
+                ]
```

Comment:
```suggestion
                scooters_to_deliver = [
                    scooter.id for scooter in vehicle.scooter_inventory[:len(vehicle.current_location.scooters)-vehicle.current_location.ideal_state]
                ]
```

Reply:
If we don't allow the policy to deliver more scooters than the ideal state..?

Reply:
Skipping this as it will screw up the rest of the policy conditioning on vehicle inventory